tests/unit/withOnyxTest.js

```diff
@@ -153,3 +153,73 @@ describe('withOnyx', () => {
             });
     });
 });
+
+describe('withOnyx', () => {
+    it('using mergeCollection to modify one item should only effect one component', () => {
+        const onRender1 = jest.fn();
+        const onRender2 = jest.fn();
+        const onRender3 = jest.fn();
+
+        // GIVEN there is a collection with three simple items in it
```

Comment:
NAB: I REALLY appreciate the `GIVEN/WHEN/THEN` docs here! It makes it so much easier to follow what's going on and what we're testing for. `:bigfan:`

Reply:
Yeah, I love that format for testing comments. @nkuoch was the one that originally showed it to me!